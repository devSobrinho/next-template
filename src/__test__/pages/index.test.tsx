import { render, screen } from '@testing-library/react'
import Home from '../../pages'

describe('Home', () => {
  it('renders a heading', () => {
    const { container } = render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /Ola mundo/i,
    })

    expect(heading).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})