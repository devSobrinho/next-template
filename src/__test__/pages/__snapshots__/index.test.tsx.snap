// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <div>
    <main>
      <h1>
        Ola mundo
      </h1>
    </main>
    <footer>
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by
         
        <span>
          <img
            alt="Vercel Logo"
            data-nimg="1"
            decoding="async"
            height="16"
            loading="lazy"
            src="/vercel.svg"
            srcset="/vercel.svg 1x, /vercel.svg 2x"
            style="color: transparent;"
            width="72"
          />
        </span>
      </a>
    </footer>
  </div>
</div>
`;
